webpack.config.js

```diff
@@ -14,11 +14,14 @@ module.exports = {
           },
         },
       }, {
-        test: /\.css$/,
-        loader: 'style-loader!css-loader',
+        test: /\.css$/i,
+        use: ['style-loader', 'css-loader'],
       }, {
         test: /\.(jpe?g|png|gif|woff|woff2|eot|ttf|svg)(\?[a-z0-9=.]+)?$/,
```

Comment:
Do you still need to transpile all these files if you have you svg saved as a path?

Reply:
Good catch, I actually don't. This was just a config I pasted in from a webpack config demo. I do still need the .png for transpiling the static modal images. But, I will clean up the rest. Thanks.